invenio_oauthclient/ext.py

```diff
@@ -94,7 +94,7 @@ def __init__(self, app):
             )
 
             # Register sign-up handlers
-            def dummy_handler(remote, *args, **kargs):
+            def dummy_handler(remote, *args, **kargs):  # pragma no cover
```

Comment:
IMHO it would be better to create simple test that registers new client without any handler.